spec/features/pictures_features_spec.rb

```diff
@@ -0,0 +1,62 @@
+feature 'Pictures' do
+  context 'no pictures have been posted' do
+    scenario 'should display a prompt to add a picture' do
+      sign_up
+      visit '/pictures'
+      expect(page).to have_content 'No pictures yet'
+      expect(page).to have_link 'Post a picture'
+    end
+  end
+
+  # context 'pictures have been added' do
+  #
+  #   before do
+  #     Picture.create(title: 'My cat is awesome')
+  #   end
+  #
+  #   scenario 'display picture titles' do
+  #     sign_up
+  #     visit '/pictures'
+  #     expect(page).to have_content('My cat is awesome')
+  #     expect(page).not_to have_content('No pictures yet')
+  #   end
+  # end
+
+  context 'creating pictures' do
+    scenario 'prompts users to fill in a form and then displays their picture' do
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)